sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_stdout_events_v1/metadata.yaml

```diff
@@ -7,9 +7,11 @@ labels:
   application: fxa
   incremental: true
   schedule: daily
-scheduling:
-  dag_name: bqetl_fxa_events
-  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
+# Query descheduled as a direct result of AWS migration to GCP
+# on 27th September 2023 the last AWS instances were spun down.
+# scheduling:
+#   dag_name: bqetl_fxa_events
```

Comment:
I don't think this should be descheduled, as the payments server logs are still being recorded here.

Reply:
I believe you are correct here. This is something I have forgotten about. I have reverted this change and added a comment. We should also make sure that the FxA team is aware this is still happening since the data is still landing in the old GCP project (in case they have plans to drop it at some point).

Reply:
I see you already reached out to the FxA team, thanks @sean-rose 🙇